_quarto.yml

```diff
@@ -107,12 +107,19 @@ website:
           - RAP/rap-support.qmd
           - RAP/rap-faq.qmd
           - RAP/rap-statistics.qmd
-      - section: "ADA and Databricks"
+      - section: "Understanding ADA and Databricks"
```

Comment:
I'm looking at this and it feels slight odd having two separate top level things for Databricks when we have other bigger categories as one.

I like the breaking out of the pages, but I'd have them all under a 'Analytical Data Access (ADA) and Databricks` heading?

![image](https://github.com/user-attachments/assets/8eb5935a-e706-4f87-a0f6-c9434cb4727b)